static/templates/draft_table_body.hbs

```diff
@@ -6,11 +6,13 @@
                 <div class="exit">
                     <span class="exit-sign">&times;</span>
                 </div>
-                <div class="removed-drafts">
+                <span class="removed-drafts">
```

Comment:
The changes in this file seem like they were accidentally added to this commit! I'm guessing they're from a merge conflict where the wrong (old code-base) code was picked.